### ISSUE TITLE
[Feature Request] Mobile app Screen Lock

### ISSUE THREAD

**Issue by danthegoodman1** [keybase/client#12576]
One thing I love about apps like Signal and Wickr is that I can have them require a passcode/TouchID/FaceID when I reopen the app, or even have them log out every time.

The Keybase Electron app has this (macOS but that's not OS specific), why can't it be implemented into the iOS/Android App? Having the code/FaceID/TouchID to access the app would be really awesome.

**Comment by nilz3ro** [comment]
## @keybase, 
I agree with @danthegoodman1. I was just talking with a few of my friends who use Keybase on iOS and Windows, it'd be great if there was a biometric login option for iPhone and a Windows Hello option for Windows 10.

**Comment by BlockLabs** [comment]
@adamjspooner @joshblum @nathunsmitty, yes please. what @danthegoodman1 && @nilz3ro said. I third this `feature request`. is *thirding* a thing ¯\_(ツ)_/¯

**Comment by bussiere** [comment]
+1 for the locked screen as signal.

For me it's just missing this feature to be awsome

**Comment by jarbro** [comment]
Would love to be able to lock the chat window as well on Mac or PC.

**Comment by Mansarde** [comment]
+1 This would be very handy, as my password is very complex and a pain to enter.

**Comment by csk** [comment]
I am also missing this feature. +1

**Comment by Seia-Soto** [comment]
+1 to this feature, just like Telegram, the secure messenger does.
